gitops/kcp/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://raw.githubusercontent.com/openshift-pipelines/pipelines-service/main/gitops/triggers/triggers-crds/base/namespace.yaml
+  - https://raw.githubusercontent.com/roming22/pipelines-service/my/gitops/gitops/kcp/base/plnsvc-manager.yaml
```

Comment:
Is providing your git repo (roming22) intentional for this PR? Would you be updating this to upstream repo later on?

Reply:
Fixed